roles/nova-data/tasks/libvirt.yml

```diff
@@ -56,29 +58,124 @@
   notify: restart nova services
   when: openstack_install_method == 'source'
 
+- name: make pki paths
+  file:
+    path: "{{ item }}"
+    state: directory
+    owner: root
+    group: kvm
+  with_items:
+    - /etc/pki/CA
+    - /etc/pki/libvirt/private
+
+- name: install CA file
+  template:
+    src: etc/pki/CA/cacert.pem.j2
+    dest: /etc/pki/CA/cacert.pem
+    owner: root
+    group: kvm
+    mode: 0644
+  register: ca_install
+  notify:
+    - set libvirt pki dir owner
+    - set libvirt pki perms
+
+- name: upload CA key
+  template:
+    src: etc/pki/CA/cacert.key.j2
+    dest: /etc/pki/CA/cacert.key
+  register: ca_key_install
```

Comment:
The files here have pretty generic names. Is there any risk of overwriting other files here? Can we name them more specific for the libvirt usage?